services/workflow_orchestrator.py

```
@@ -178,15 +178,19 @@ def _record_ceo_fallback(reason: str) -> None:
+    Thin wrapper over the shared :func:`brain_policy.allow_paid_brain` so the
+    orchestrator and the ``agent/loop.py`` runtime share one source of truth
+    (issue #656). Default ``False`` — the brain never silently calls a paid API;
+    when no free provider is configured it falls through to local Ollama.
     """
-    return os.environ.get("ALLOW_PAID_BRAIN", "").strip().lower() in {
-        "1", "true", "yes", "on",
-    }
+    try:
+        from brain_policy import allow_paid_brain
+        return allow_paid_brain()
+    except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)